docs/source/api/tracker.md

```diff
@@ -86,21 +86,42 @@ a `Tracker.withComputation` call.
 
 ```javascript
 Tracker.autorun(async function example1(computation) {
-  let asyncData = 
-    await Tracker.withComputation(computation, () => asyncDataFunction());
-  let users = Meteor.users.find({}).fetch();
+  let asyncData = await asyncDataFunction();
+  let users =
+    await Tracker.withComputation(computation, () => Meteor.users.find({}).fetch());
 });
 ```
 > If you want to get computation in other way you can use `Tracker.currentComputation`
 
-#### Using async callbacks in versions of Meteor prior to 2.10
+
+{% apibox "Tracker.withComputation" %}
+
+In general, the rules to use `Tracker.withComputation` like this:
+1. `async` function *before the first* `await` if just like a sync one.
+2. `async` function *after the first* `await` looses the `Tracker.currentComputation`
+but it can be restored using `Tracker.withComputation`, *but only within the callback*.
+
+If you have for example:
+
+```javascript
+Tracker.autorun(async function example1(computation) {
+  let asyncData = await asyncDataFunction();
+  let users = await Meteor.users.find({}).fetch(); // looses the computation
+  let users2 = 
+    await Tracker.withComputation(computation, () => Meteor.users.find({}).fetch()); // maintains the computation
```

Comment:
I'd repeat what is said below, i.e., "looses the computation" means "won't trigger reruns", as that's _the_ most important part of it.